### PR TITLE
Automation for MAGN-7146 

### PR DESCRIPTION
**Pull request by monikaprabhu**
Add automation test to check .dyn file loads as Manual if reopen after a crash.
// MAGN-7146 was saved after a VM crash.

### CODE REVIEW

**Comment by monikaprabhu** [comment]
@junmendoza - please review and merge

**Comment by monikaprabhu** [comment]
@junmendoza - I've added code to fail the test if it regresses.

File: test/DynamoCoreTests/CrashProtectionTests.cs [hunk 1]
```diff
@@ -147,5 +147,17 @@ public void FlagIsSetToTrueAfterRunSuccessfullyCompletes()
             }
         }
 
+        [Test]
+        public void MAGN_7146()
```
Comment (junmendoza) [inline test/DynamoCoreTests/CrashProtectionTests.cs]: @monikaprabhu: Thanks for adding. Just note that this behavior will work for any file with the Manula Flag set, not just VM related files

**Comment by junmendoza** [comment]
LGTM

**Pull request merged**